examples/custom-usage-counter/custom_usage_counter.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+import argparse
+import asyncio
+
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from decimal import Decimal
+from pathlib import Path
+
+from yapapi.strategy import LeastExpensiveLinearPayuMS, DecreaseScoreForUnconfirmedAgreement
+
+from yapapi.log import enable_default_logger
+
+from yapapi.props.base import prop, constraint
+from yapapi.props import inf, com
+
+from yapapi import Golem
+from yapapi.ctx import ActivityUsage
+from yapapi.payload import Payload
+from yapapi.services import Service, ServiceState
+
+
+@dataclass
+class CustomCounterServicePayload(Payload):
+    runtime: str = constraint(inf.INF_RUNTIME_NAME, default="test-counters")
+    min_mem_gib: float = constraint(inf.INF_MEM, operator=">=", default=0.5)
```

Comment:
do we need those additional constraints here? (besides the `runtime` ?)

Reply:
I don't know, all examples contained `min_mem_gib` and `min_storage_gib`.

Reply:
I'll remove them.

Reply:
Done